### ISSUE TITLE
Rule ordering in generated css

### ISSUE THREAD

**Issue by mrmcc3** [mrmcc3/tailwind-clj#2]
At the moment rules are sorted by class name before writing the css this is most likely a mistake. should investigate how tailwind orders css output

**Comment by retro** [comment]
From my research it seems that the order of classes for one media query are not important, but it's important to order media query groups (to ensure correct precedence of CSS rules). So something like:

1. default classes
2. sm classes
3. md classes

BTW, this looks like a great project, I was about to start implementing something very similar my self. Are you looking for contributors?

**Comment by mrmcc3** [comment]
Yep that matches my understanding of how the classes should be ordered. I think the current behaviour spits the media query groups out in order, the classes within each group are just sorted by name.

Since I wrote this there's been a fair bit of change in the tailwind space. 
- new releases of tailwindcss with new utilities.
- plugins. https://tailwindcss-custom-forms.netlify.com/. would be nice to have something like this 
- tailwindUI. I think that uses a custom tailwind plugin

Anyway I'd be happy to accept contributions
